modules/navier_stokes/doc/content/source/linearfvbcs/LinearFVP1RadiationMarshakBC.md

```diff
@@ -0,0 +1,27 @@
+# LinearFVP1RadiationMarshakBC
+
+This boundary condition implements the Marshak boundary conditions for
+the radiative heat flux.
+
+## Overview
+
+The Marshak boundary condition implemented reads as follows [!citep](modest2021):
+\begin{equation}
+(-\Gamma \nabla G) \cdot \vec{n_b}   = (G - 4 \sigma T_{b, rad}^4) \frac{\epsilon_b}{2 \Gamma (2 - \epsilon_b)}
+\end{equation}
+
+where:
+
+- $F_G$ is the normal space derivative of the radiation heat flux at the boundaries defined by the `boundary` parameter
+- $G$ is the radiative heat flux (SI units (W/m$^2$))
+- $\sigma$ is the Stefan-Boltzmann constant (SI units (W/m$^2$/K$^4$))
+-  $\sigma_a$ is the absorption coefficient (SI units (1/m))
```

Comment:
1/m so it might be a cross section in a way?

Reply:
it is like an attenuation coefficient